Xamarin.Forms.Core/Frame.cs

```diff
@@ -11,6 +11,8 @@ public class Frame : ContentView, IElementConfiguration<Frame>
 
 		public static readonly BindableProperty HasShadowProperty = BindableProperty.Create("HasShadow", typeof(bool), typeof(Frame), true);
 
+		public static readonly BindableProperty CornerRadiusProperty = BindableProperty.Create(nameof(CornerRadius), typeof(float), typeof(Frame), -1.0f);
```

Comment:
what's the meaning of "-1" ?
- don't use a corner radius, or
- use whatever default on the platform